src/zeroband/inference/genesys/ascii_tree_formatting.py

```diff
@@ -0,0 +1,48 @@
+from typing import Dict
+import re
+import difflib
+
+
+def compute_reward(completion: str, verification_info: Dict):
+    # Extract answer
```

Comment:
can u add a short docstring explaning how the rewards is computed ? No need to be super specific tho